gremlin-integrations/templates/secret-ssl-cert-file.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.ssl.certFile }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ssl-cert-file
```

Comment:
Do we need to change the name to avoid collision with gremlin daemon installed certs?

Reply:
Usually it's the same certificate that is used when connecting. But of course we can separate them in order not to create a confusion.

Reply:
yeah, it's not guaranteed to be the same, probably safer for now